src/components/BlogForm/StoredArrays.tsx

```diff
@@ -0,0 +1,18 @@
+import { TypographyProps } from "@mui/material";
```

Comment:
make this a .ts file and change the name to something more descriptive

also update the MenuItems to menuItems since its not a React component